app/src/main/java/com/hedvig/app/feature/chat/viewmodel/UserViewModel.kt

```diff
@@ -42,15 +46,25 @@ class UserViewModel(
         }
     }
 
-    fun logout(callback: () -> Unit) {
+    fun logout(context: Context, callback: () -> Unit) {
```

Comment:
While we're at it - should we make this a `suspend fun` instead of being callback-based?

Also, should we remove the dependency on `context` for the caller, and instead allow `LoginStatusService` to handle the deletion of the `authenticationToken`?

Reply:
Created a LogoutUseCase and removed the callback